docs/ceph-csi-upgrade.md

```diff
@@ -303,4 +304,13 @@ For each node:
   - The pod deletion causes the pods to be restarted and updated automatically
     on the node.
 
-we have successfully upgrade RBD csi from v1.2.2 to v2.0.0
+we have successfully upgraded RBD csi from v1.2.2 to v2.0.0
+
+##### 5.0 Handling the node reboot issue for CephFS and RBD pvc
+
+If you have mounted the rbd/cephfs PVC on nodes by older cephcsi drivers, you
```

Comment:
Some rewording:

With prior versions of ceph-csi a node reboot may hang the reboot process. This is due to existing cephfs or rbd network mounts not carrying the `_netdev` mount option. The missing mount option is fixed, and future mounts using ceph-csi would automatically carry the required flags needed for a clean reboot.

It is suggested to upgrade to the latest ceph-cei release and drain application pods on all the nodes so that new mount option `_netdev` can be adding to all the mountpoints.